prac_06/car.py

```diff
@@ -28,4 +29,10 @@ def drive(self, distance):
         else:
             self.fuel -= distance
         self.odometer += distance
-        return distance
\ No newline at end of file
+        return distance
+
+    def __str__(self):
+        fuel = self.fuel
+        odometer = self.odometer
+        name = self.name
```

Comment:
Declaring these variables is unnecessary
Consider using self.fuel instead of fuel etc.
return "{} has fuel={}, odometer={} ".format(self.name, self.fuel, self.odometer)